_data/standard/html5.yml

```diff
@@ -32,10 +32,12 @@ administrations:
       created: '2018-01-23'
       metadataLastUpdated: '2018-11-15'
     references:
-      - URL:
```

Comment:
There can be more than one reference.  See schema https://canada-ca.github.io/ore-ero/en/open-standard-schema.html